test/index.js

```diff
@@ -2700,6 +2700,70 @@ test('escape', (t) => {
     'should support empty `join`, `handlers`, `extensions` in an extension (coverage)'
   )
 
+
+  t.equal(
+    to(
+      {
+        "type": "root",
+        "children": [
+            {
+                "type": "list",
+                "ordered": true,
+                "start": 1,
+                "spread": false,
+                "children": [
+                    {
+                        "type": "listItem",
+                        "spread": true,
+                        "checked": null,
+                        "children": [
+                            {
+                                "type": "paragraph",
+                                "children": [
+                                    {
+                                        "type": "text",
+                                        "value": "foo"
+                                    }
+                                ]
+                            },
+                            {
+                                "type": "list",
+                                "ordered": false,
+                                "start": null,
+                                "spread": false,
+                                "children": [
+                                    {
+                                        "type": "listItem",
+                                        "spread": false,
+                                        "checked": null,
+                                        "children": [
+                                            {
+                                                "type": "paragraph",
+                                                "children": [
+                                                    {
+                                                        "type": "text",
+                                                        "value": "bar"
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    },
+      {
+        join: [() => 0],
+      }
+    ),
+    '1.  foo\n    *   bar\n',
+    'should make `join` from options highest priority'
+  )
```

Comment:
Looks like some of the styling does not match this project. Can you run `npm test` locally for format everything?